### PR TITLE
Feature/upstream merge

### PR DESCRIPTION
**Pull request by j-mie**
🆑 JamieH - Ports from /tg/
fix: Ports some bug fixes
rscadd: All mobs can now *spin!
rscadd: Cyborgs now have handholds. This means you can ride around on them, but if you get stunned or hit, you fall off! The cyborg can also throw you off by spinning
rscadd: Adds the smite button for admins
rscadd: You can now emag the escape pods to launch them
tweak: APCs are now immune to fire
rscadd: Skewium: It'll fuck you up
/🆑

Thanks /tg/ <3

### CODE REVIEW

File: code/game/atoms_movable.dm [hunk 1]
```diff
@@ -160,10 +159,10 @@
 			if(destarea && old_area != destarea)
 				destarea.Entered(src, oldloc)
 
-		for(var/atom/movable/AM in destination)
-			if(AM == src)
-				continue
-			AM.Crossed(src)
+				for(var/atom/movable/AM in destination)
```
Comment (ghost) [inline code/game/atoms_movable.dm]: Good job 👏

**Review by ghost — CHANGES_REQUESTED** [review]

File: code/game/atoms_movable.dm (hunk 1 above)
Comment (tortellinitony) [reply code/game/atoms_movable.dm]: What?

Comment (ghost) [reply code/game/atoms_movable.dm]: https://github.com/HippieStation/HippieStation13/pull/183/files/9a9872a3f1f280eeb780e45bb612875bb31c455a#diff-85b909bea63591ebec264bda09fbb017L142 look at this

Comment (j-mie) [reply code/game/atoms_movable.dm]: Fixed.

**Review by tortellinitony — APPROVED** [review]

**Pull request merged**